packages/circuit-ui/components/Theming/Theming.docs.mdx

```diff
@@ -2,181 +2,92 @@ import { Story } from '../../../../.storybook/components';
 
 # Theming
 
-This is working documentation for our theming initiative.
+Circuit UI supports color theming. It uses semantic color tokens, which can be customized to deviate from the default light theme.
 
-## Semantic color naming
+## Semantic color tokens
 
-- **usage (in UI)**: background, foreground, border _(, icon)_
-- **role (semantic meaning)**: neutral, accent, confirm, notify, alert _(, promo, ...)_
-- **variant**: default, subtle, highlight _(, ...)_
-- **state**: idle, hover, active
+At the core of Circuit UI theming are semantic color tokens. Instead of referencing color values (as hex codes, for example), Circuit UI components reference semantic color names, which are mapped to a color value depending on te theme.
```

Comment:
```suggestion
At the core of Circuit UI theming are semantic color tokens. Instead of referencing color values (as hex codes, for example), Circuit UI components reference semantic color names, which are mapped to a color value depending on the theme.
```